src/main.py

```diff
@@ -633,6 +635,8 @@ def main(args):
                 qm_all_path = config.WORKING_DIR + "/ALC-" + repr(THIS_ALC-1) + "/DFTB-all/"
             elif config.IGAS_QM_METHOD == "Gaussian":
                 qm_all_path = config.WORKING_DIR + "/ALC-" + repr(THIS_ALC-1) + "/GAUS-all/"
+            #elif config.IGAS_QM_METHOD == "QE":
```

Comment:
I'm going to hold off on merging this PR because there seem to be edits (e.g., this line) arising from hacking in QE that would break functionality (e.g., for VASP).